includes/discovery/sensors/current/adva.inc.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * LibreNMS - ADVA device support - Current
+ *
+ * @category   Network_Monitoring
+ * @package    LibreNMS
+ * @subpackage ADVA device support
+ * @author     Christoph Zilian <czilian@hotmail.com>
+ * @license    http://gnu.org/copyleft/gpl.html GNU GPL
+ * @link       https://github.com/librenms/librenms/
+
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ **/
+
+// ******************************************
+// ***** Sensors for ADVA FSP150EG-X Chassis
+// ******************************************
+
+if (starts_with($device['sysObjectID'], 'enterprises.2544.1.12.1.1')) {
```

Comment:
If this is to restrict the check to this OS then you can remove this if check as we only run sensors with the same filename as the os.